src/Definition/Exasol.php

```diff
@@ -407,4 +408,32 @@ public function getBasetype(): string
         }
         return $basetype;
     }
+
+    public static function createByBasetype(string $basetype): self
+    {
+        $basetype = strtoupper($basetype);
+
+        if (!BaseType::isValid($basetype)) {
+            throw new InvalidTypeException(sprintf('Base type "%s" is not valid.', $basetype));
+        }
+
+        switch ($basetype) {
+            case BaseType::BOOLEAN:
+                return new self(self::TYPE_BOOLEAN);
+            case BaseType::DATE:
+                return new self(self::TYPE_DATE);
+            case BaseType::FLOAT:
+                return new self(self::TYPE_DOUBLE_PRECISION);
+            case BaseType::INTEGER:
+                return new self(self::TYPE_INTEGER);
+            case BaseType::NUMERIC:
+                return new self(self::TYPE_DECIMAL, ['length' => '36,36']);
```

Comment:
mozna by to nemuselo vracet celou instanci (kvuli defaulltni delce), ale jen type jako string?
Jak na tim znova premyslim, tak asi bude lepsi vracet jen ten string...

Reply:
puvodne jsem si taky myslel, ze bude fajn vracet string, ale ten UC, ktery mame na stole rika, ze chceme instanci, ne si to pak rucne tvorit v connection. Takze bych to nechal tak

Jen prosim na ty delky pouzij vyse uvedene konstanty. Zrovna ten numeric jsem nedavno upravoval

Reply:
Tyhle hodnoty byly z toho kodu UI a prislo mi to jako rozumny, mit to stejny. Ale zase kdyz mame "svoje" defaulty, tak OK.